macaroonbakery/tests/test_time.py

```diff
@@ -10,7 +10,8 @@
 
 import macaroonbakery.checkers as checkers
 
-t1 = pyrfc3339.parse('2017-10-26T16:19:47.441402074Z')
+t1 = pyrfc3339.parse(
+    '2017-10-26T16:19:47.441402074Z', utc=True).replace(tzinfo=None)
```

Comment:
In this case, because we know the time is in UTC, we can actually use produce_naive=True.

Reply:
done